app/components/item/Block.vue

```diff
@@ -17,6 +17,7 @@
       </h5>
       <div v-if="item.description" class="item-text mt-2">
         <div v-sanitize="item.description" />
+        Hello
```

Comment:
What's this?

Reply:
OOO, sorry, it was left there by accident :-(